helpers/scraper.js

```diff
@@ -0,0 +1,366 @@
+        ['Nevada', 'NV'],
+        ['New Hampshire', 'NH'],
+        ['New Jersey', 'NJ'],
+        ['New Mexico', 'NM'],
+        ['New York', 'NY'],
+        ['North Carolina', 'NC'],
+        ['North Dakota', 'ND'],
+        ['Ohio', 'OH'],
+        ['Oklahoma', 'OK'],
+        ['Oregon', 'OR'],
+        ['Pennsylvania', 'PA'],
+        ['Rhode Island', 'RI'],
+        ['South Carolina', 'SC'],
+        ['South Dakota', 'SD'],
+        ['Tennessee', 'TN'],
+        ['Texas', 'TX'],
+        ['Utah', 'UT'],
+        ['Vermont', 'VT'],
+        ['Virginia', 'VA'],
+        ['Washington', 'WA'], 
+        ['District of Columbia', 'DC'],
+        ['West Virginia', 'WV'],
+        ['Wisconsin', 'WI'],
+        ['Wyoming', 'WY'],
+    ];
+
+    for (var i = 0; i < states.length; i++) {
+      if (state == states[i][0]) {
+        return states[i][1];
+      }
+    }
+    return "";
+}
+
+const getCalifornia = function(school) {
+  let splitSchool = school.split(" ");
+  let usableWords = [];
+  for (var i = 0; i < splitSchool.length; i++) {
+      if (splitSchool[i].includes("!")) {
+        break;
+      } else {
+        usableWords.push(splitSchool[i]);
+      }
+    } 
+    if (school.includes("Los Angeles")) {
+      return "UCLA";
+    }
+  if (school.includes("Berkeley")) {
+    return "California";
+  } if (school.includes("University of California")) {
+      usableWords[0] = "UC";
+      return usableWords.join("_");
+    }
+   if (school.includes("California State University")) {
+      if (
+        splitSchool[splitSchool.length - 1] == "Fresno" || 
+        splitSchool[splitSchool.length - 1] == "Sacramento" || 
+        splitSchool[splitSchool.length - 1] == "Long Beach"
+        ) {
+        return splitSchool[splitSchool.length - 1].concat("_State");
+      } else {
+        return "Cal_St_".concat(splitSchool[splitSchool.length - 1]);
+      }
+  }
+}
+
+const getNC = function (school) {
+    let splitSchool = school.split(" ");
+    if (school.includes("Chapel Hill")) {
+      return "North Carolina";
+    } else {
+        return "UNC_".concat(splitSchool[2]);
+    }
+}
+
+const getWisco = function(school) {
+  let splitSchool = school.split(" ");
+  if (school.includes("Madison")) {
+    return "Wisconsin";
+  } else {
+    return "Wis_".concat(splitSchool[1]);
+  }
+}
+
+const getTexas = function(school) {
+  let splitSchool = school.split(" ");
+  let usableWords = [];
+  for (var i = 0; i < splitSchool.length; i++) {
+      if (splitSchool[i].includes("!")) {
+        break;
+      } else {
+        usableWords.push(splitSchool[i]);
+      }
+    } 
+  if (school.includes("Austin")) {
+    return "Texas";
+  } else if (school.includes("El Paso")) {
+    return "UTEP";
+  } else {
+      return "UT_".concat(usableWords.splice(1, usableWords.length - 1).join("_"));
+  }
+}
+
+const getTeamName = function(school) {
+  let usableWords = [];
+  let splitSchool = school.split(" ");
+  if (school == "Boston University") {
```

Comment:
javascript has a switch statement that might make this cleaner...this is fine by me though